mantidproject/mantid
# ILL sans IDFs: update D33, add D11 and D22

IDF Updates/changes :
- pixel_rectangular_horizontal defined but not used (D33)
- change of detector IDs (D33)
- change width of detector (122 pixels) (D22)
- add factor to account for different resolutions in x and y direction (D11)

Update of D33 due to generator python script updates (readability, maintenance):
https://github.com/mantidproject/mantidgeometry/issues/102

**To test:**

All tests should pass.

Check if the instruments are correctly defined:

D33: https://www.ill.eu/instruments-support/instruments-groups/instruments/d33/characteristics/
D22: https://www.ill.eu/instruments-support/instruments-groups/instruments/d22/characteristics/
D11: https://www.ill.eu/instruments-support/instruments-groups/instruments/d11/characteristics/

The instrument responsible wrote:

The rear detector of D33 has  256x128 pixels with 2.5x5 mm (in horizontal x vertical direction) as only standard resolution. The only SANS instrument, which changes its resolution on a standard basis is D11 (either 256^2 or 128^2) for now.

The front side panels framing the main detector (#0) of D33 have 32x256 pixels with 5x2.5 mm in (height/length). Actually, the 5mm resolution is a physical limit giving by the width of a single He tube, along the detection wire, one is rather free to choose the discrimination, here choosen to be 2.5mm with 256 pixels. If you want to keep a similiar labeling as is Grasp, it will be #1: Right, #2: Left, #3: Bottom, #4: Top (looking from the detector towards the sample). I assume that life is easier, when there is an identifier for the detectors. In case you want to e.g. only use the signal of the main detector for analysis. 

Fixes #21759.

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
